README.md

```diff
@@ -0,0 +1,3 @@
+The filter order dependency resolver accepts dependency declarations in
+the form of injected beans and reorders servlet filters in order to
+accord to the desired dependencies.
```

Comment:
I prefer basic usage in README, just some examples to see how it's used.

Reply:
I thought about including that.  But it's also already in the Javadocs.  Duplicating it would be worse, so we just ended up with a pointer in the readme to check out the Javadoc.  What do you think?  Maybe literally just some code to emit a bean would really give you an idea of the payoff?  What do you think?

Reply:
What do you think of dc0482af288791a74be0a092ee3fb57cc481e8c6 @adamzr ?